src/ziggurat/metrics.clj

```diff
@@ -56,8 +56,10 @@
 
 (defn- inc-or-dec-count
   ([sign metric-namespace metric]
-   (inc-or-dec-count sign metric-namespace metric nil))
-  ([sign metric-namespaces metric additional-tags]
+   (inc-or-dec-count sign metric-namespace metric 1 {}))
+  ([sign metric-namespace metric n]
```

Comment:
you changed the contract here, but did not change the function call in 
```
(defn multi-ns-increment-count [nss metric additional-tags]
  (doseq [ns nss]
    (increment-count ns metric additional-tags)))
```
😞

Reply:
you'll have to change it everywhere it is being called from

Reply:
the case is not handled here where users pass `additional-tags` but not `n`

Reply:
done